src/features/slackapi/slackapi-utils.ts

```diff
@@ -54,17 +56,20 @@ namespace SlackApiUtilities {
 Hi ${firstName},
 ${numberOfToday === 1 ? "Last friday" :"Yesterday"} (${displayDate}) you worked ${logged} with an expected time of ${expected}.
 ${message}
-Project time: ${project}, Internal time: ${internal}.
-Your percentage of billable hours was: ${billableHoursPercentage}% ${+billableHoursPercentage >= 75 ? ":+1:" : ":-1:"}
+Logged project time: ${loggedProject}, Billable project time: ${billableProject}, Non billable project time: ${nonBillableProject}, Internal time: ${internal}.
+Your percentage of billable hours was: ${billableHoursPercentage}% ${parseInt(billableHoursPercentage) >= minimumBillableRate ? ":+1:" : ":-1:"}
```

Comment:
MinimumBillable rate is coming from timebank, this was previously set at 75% for everyone, it should be confirmed which is the appropriate value.